chunky/src/java/se/llbit/chunky/renderer/postprecessing/PostProcessingFilters.java

```diff
@@ -0,0 +1,44 @@
+package se.llbit.chunky.renderer.postprecessing;
+
+import se.llbit.chunky.plugin.PluginApi;
+
+import java.util.*;
+
+public abstract class PostProcessingFilters {
+
+  private static final Map<String, PostProcessingFilter> filters = new HashMap<>();
+  // using tree map for the added benefit of sorting by name
+  private static final Map<String, PostProcessingFilter> filtersByName = new TreeMap<>();
+  private static final PostProcessingFilter defaultFilter;
+  
+  static {
+    defaultFilter = new GammaCorrectionFilter();
+    addPostProcessingFilter(defaultFilter);
+    addPostProcessingFilter(new NoneFilter());
+    addPostProcessingFilter(new Tonemap1Filter());
+    addPostProcessingFilter(new ACESFilmicFilter());
+    addPostProcessingFilter(new HableToneMappingFilter());
+  }
+  
+  public static Optional<PostProcessingFilter> getPostProcessingFilterFromId(String id) {
+    return Optional.ofNullable(filters.get(id));
+  }
+
+  public static Optional<PostProcessingFilter> getPostProcessingFilterFromName(String name) {
+    return Optional.ofNullable(filtersByName.get(name));
+  }
```

Comment:
Please `@Deprecated` and `// TODO` this. I know it's required for now but at some point we should only use IDs to identify and get dynamically registered things. This requires a new Combobox that supports IDs _and_ display names so is for another PR.